src/lib.rs

```diff
@@ -62,6 +62,9 @@ mod serde_utils;
 mod endian;
 // re-export bitcoin deps which we re-use
 pub use bitcoin::{bech32, hashes};
+// re-export bitcoin locktime types, which have identical semantics as in Elements,
+//  and differ only by what kind of transaction they appear in.
+pub use bitcoin::{blockdata::locktime, LockTime, PackedLockTime, Sequence};
```

Comment:
Would need to double-check that having different block rates(1min vs 10 mins) does not mess up the APIs here. I would really like to have separate `Sequence` and `LockTime` types here because we can possibly change semantics to mean different things. Initially, we used bitcoin::Script here but found it better to use elements-specific types in the long run.

Reply:
Ok, I'll make them distinct types. I agree that Elements blockheights are semantically different from Bitcoin blockheaights.

Would it be OK if I just did newtypes around the rust-bitcoin types, and passed through every method?

Reply:
I think we should copy the lock time and sequence module from rust-bitcoin instead of wrapping new types. Wrapper types might make the API annoying to use. The modules are self-contained and hopefully should be easy to copy-paste. 

We did the same thing for `Script` and even decided it was for keys. But we lose the benefit of gaining bug fixes and updates from upstream for free.

Reply:
Overall, I think defining all primitives in `rust-elements` will make things easy in the future as 
1) I don't expect the upstream locktime/sequence modules to change often
2) When we have the CSV  for relative timelocks in blocks fixed in elements/liquid, we can change things here.